course/frontend/arrays/scripts/main.js

```diff
@@ -213,3 +213,76 @@ function flat(arr, newArray) {
 
 flat([1, 2, 3, [4, 5], 6]);
 flat([1, 2, [3, 4, [1, 2, [1, 2, 3]]], 5, 6]);
+
+
+// Exercises 28, 29
+
+/*
+### unionArray
+
+Write a JavaScript program to compute the union of two arrays.
+Sample Data :
+
+    console.log(unionArray([1, 2, 3], [100, 2, 1, 10]));
+    [1, 2, 3, 10, 100]
+
+### removeFalsy
+
+Write a JavaScript function to remove. 'null', '0', '""', 'false', 'undefined' and 'NaN' values from an array.
+
+    Sample array : [NaN, 0, 15, false, -22, '',undefined, 47, null]
+    Expected result : [15, -22, 47]
+
+*/
+
+/** 
+ * Function unionArray 
+ * Returns one numeric merged array without duplicated values.
+ * @param {array} first argument
+ * @param {array} second argument
+ * @returns {array} numeric values.
+ */
```

Comment:
EACH function should be tied to its doc and implementation. so, following this rule i encourage you to split it into two, that is:

/**
 * DOC
 */
var whatever;
(function() {
// impl of whatever here
})();

/**
 * DOC 2
 */
var anotherThing;
(function() {
// impl of anotherThing here
})();

the reason why doing so is as good practice for documentation (puting the doc tied to the implementation) and splitting the implementation too, so later on you can easily move each one to its own file, independently ,)

SAME with tests (console.logs in this case)